packages/analyzer/fixtures/plugin-lit/fixture/custom-elements.json

```diff
@@ -35,6 +35,18 @@
                 "text": "array"
               }
             },
+            {
```

Comment:
TODO: make sure that other lit-created attributes get `attribute: "name"` too, not only the reflecting ones

Reply:
The same goes for `@attr` JSDoc for vanilla